packs/vsphere/actions/vm_hw_cpu_mem_flex.yaml

```diff
@@ -0,0 +1,23 @@
+---
+  name: "vm_hw_cpu_mem_flex"
```

Comment:
flex is just the same as edit (increase or decrease). I can rename as edit if that's better.